atomic_reactor/plugins/post_rpmqa.py

```diff
@@ -43,10 +32,13 @@ def __init__(self, tasker, workflow, image_id, ignore_autogenerated_gpg_keys=Tru
         self.ignore_autogenerated_gpg_keys = ignore_autogenerated_gpg_keys
 
     def run(self):
-        fmt = self.sep.join(["%%{%s}" % tag for tag in self.rpm_tags])
+        # If another component has already filled in the image component list, skip
+        if self.workflow.image_components is not None:
+            return None
```

Comment:
This line doesn't seem to get test coverage.

Reply:
Fixed.